### PR TITLE
Post: "Configuring ESXi 6 on Hetzner".

### PR DESCRIPTION
**Pull request by nickcharlton**

### CODE REVIEW

**Comment by danielgroves** [comment]
So far it sounds good. You make it sound easy which is probably a good thing. As someone who isn't very familiar ESXi I do wonder if further screenshots might help clarify exactly what people should be looking for. Obviously you still have a few placeholders at the moment. I'll add a couple more comments in the file changes.

File: _posts/2016-09-16-configuring-esxi-6-on-hetzner.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+2. Create another vSwitch and Port Group (perhaps, `subnet1`).
+3. Add another network adapter to the router VM.
+4. Connect this new network adapter to the new vSwitch.
+5. Configure VMs in the same way.
+
+You should now be able to bring up additional VMs and use the rest of your
+assigned subnet!
+
+[Hetzner]: https://hetzner.de
+[VMware ESXi]: http://www.vmware.com/products/esxi-and-esx.hml
```
Comment (danielgroves) [inline _posts/2016-09-16-configuring-esxi-6-on-hetzner.md]: This link seems to 404.

Comment (nickcharlton) [inline _posts/2016-09-16-configuring-esxi-6-on-hetzner.md]: Fixed.

**Comment by danielgroves** [comment]
Sounds good to me. You're good at making complex subjects sounds simple; I've not seen the Hetzner Robot control panel, but I presume the points you've mentioned are fairly obvious?

**Comment by nickcharlton** [comment]
@danielgroves As obvious as a not-amazing UI can make it.

**Pull request merged**